tracer/test/test-applications/security/Samples.Security.AspNetCore5/Controllers/IastController.cs

```diff
@@ -756,6 +749,58 @@ public ActionResult XpathInjection(string user, string value)
                 Content($"User " + result.ChildNodes[0].InnerText + " successfully logged.");
         }
 
+        [HttpGet("CustomAttribute")]
+        [Route("CustomAttribute")]
+        public ActionResult CustomAttribute(string userName)
+        {
+            string result = GetCustomString(userName);
+            return Content(result, "text/html");
+        }
+
+        [Datadog.Trace.Annotations.Trace(OperationName = "span.custom.attribute", ResourceName = "IastController.GetCustomString")]
+        public string GetCustomString(string userName)
+        {
+            string result = string.Empty;
+            try
+            {
+                var taintedQuery = "SELECT Surname from Persons where name = '" + userName + "'";
+                result = new SQLiteCommand(taintedQuery, DbConnection).ExecuteScalar()?.ToString();
+            }
+            catch
+            {
+                result = "Error in query.";
+            }
+            return result;
+        }
+
+        [HttpGet("CustomManual")]
+        [Route("CustomManual")]
+        public ActionResult CustomManual(string userName)
+        {
+            string result = string.Empty;
+            try
+            {
+                using (var parentScope =Datadog.Trace.Tracer.Instance.StartActive("span.custom.manual"))
```

Comment:
I strongly suggest you don't call `Datadog.Trace.Tracer.Instance` directly - we need to stop doing this to resolve various other CI issues, and we have helpers for doing this "indirectly", `SampleHelpers.CreateScope()` etc

Reply:
Done